catalyst_voices/uikit_example/lib/examples/voices_headers_examples.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:flutter/material.dart';
+
+class VoicesHeadersExamples extends StatelessWidget {
+  static const String route = '/headers-example';
+
+  const VoicesHeadersExamples({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Headers')),
+      body: SizedBox(
+        width: 360,
+        child: Column(
+          children: <Widget>[
```

Comment:
`<Widget>` type not needed probably

Reply:
Actually its because of this warning. Which is annoying but i'm not sure if it can be done better with `separatedBy` ext.

`The argument type 'SizedBox' can't be assigned to the parameter type 'SectionHeader'. (Documentation)`